optimism2/ovm2/get_contracts/insert_project_name_mapping.sql

```diff
@@ -50,6 +50,11 @@ Layerzero	Layer Zero
 Xtoken	xToken
 Instadapp	InstaDapp
 Lifi	LiFi
+Optimistic Exporer	Optimistic Explorer - Get Started NFT
+ironbank	Iron Bank
+Ironbank	Iron Bank
```

Comment:
postgres seems to treat line 54 & 55 as the same -- likely not case sensitive, leading to duplicates and failed load. i had to remove line 55 to run. will this work?

Reply:
i ran if you wanted to check output